docs/dapps/events.md

```diff
@@ -0,0 +1,184 @@
+---
+sidebar_position: 27
+title: Events
+sidebar_label: Events
+---
+
+Events are immutable, verifiable objects emitted by the smart
+contracts, stored and served by the Alephium full node. They play a
+critical role in facilitating efficient and transparent communication
+between smart contracts and off-chain applications.
+
+There are many use cases for events: DEX can use events to keep track
+of all the swaps happened in a token pair. NFT marketplace can use
+events to store all the NFT listings. Oracle can use events to signal
+requests of certain offchain info from smart contracts. Bridge can use
+events to represent certain actions that require consensus from all
+the bridge operators, etc.
+
```

Comment:
Let's document the configuration of full node events since it's disabled by default for mainnet nodes. Developers need to enable it manually.

Reply:
Fixed :+1: